src/frequenz/sdk/data_ingestion/resampling/subscriptions.py

```diff
@@ -0,0 +1,40 @@
+"""
+Common model definitions used for resampling purposes.
+
+Copyright
+Copyright © 2022 Frequenz Energy-as-a-Service GmbH
+
+License
+MIT
+"""
+from dataclasses import dataclass
+from datetime import datetime
+from typing import Optional
+
+from frequenz.channels import Receiver, Sender
+
+from frequenz.sdk.core.timeseries import Sample, TimeSeriesId
+
+
+@dataclass(frozen=True)
+class Subscription:
+    """Subscription for receiving data for a specific time series id."""
+
+    time_series_id: TimeSeriesId
+    start_time: Optional[datetime] = None
+
+
+@dataclass(frozen=True)
+class SubscriptionResponse:
+    """Response to a subscription request with a receiver for ingesting data."""
+
+    receiver: Receiver[Optional[Sample]]
+    subscription: Subscription
+
+
+@dataclass(frozen=True)
+class SubscriptionRequest:
```

Comment:
I would put this in the actor module, as it is only intended to be used with the actor.

Reply:
Makes sense, but it will probably be replaced by whatever `DataSourcingActor` introduces so I the question is if I should wait for that or move it now?

Reply:
Let's wait :+1: